Modules/Core/Common/include/itkMacro.h

```diff
@@ -355,12 +355,16 @@ namespace itk
 // prohibits the use of copy/move construction and copy/move assignment
 // functions.
 //
-#define ITK_DISALLOW_COPY_AND_ASSIGN(TypeName)                                                                         \
+#define ITK_DISALLOW_COPY_AND_MOVE(TypeName)                                                                           \
   TypeName(const TypeName &) = delete;                                                                                 \
   TypeName & operator=(const TypeName &) = delete;                                                                     \
   TypeName(TypeName &&) = delete;                                                                                      \
   TypeName & operator=(TypeName &&) = delete
 
+#if !defined(ITK_FUTURE_LEGACY_REMOVE)
+#  define ITK_DISALLOW_COPY_AND_ASSIGN(TypeName) ITK_DISALLOW_COPY_AND_MOVE(TypeName)
+#endif
+
```

Comment:
This is the most relevant part of the pull request. Please check!

Reply:
Thanks for pointing this out, it is hard to find it in GitHub interface of all the changes. Possibly make this change to `itkMacro.h` a separate first commit, with the second commit being a style commit updating from `ITK_DISALLOW_COPY_AND_ASSIGN` to `ITK_DISALLOW_COPY_AND_MOVE`.